### ISSUE TITLE
Fiddle doesn't recognize newly downloaded Electron version

### ISSUE THREAD

**Issue by treysis** [electron/fiddle#1279]
If you download a new/different Electron version from the dropdown menu, it won't be usable until after restarting the Fiddle application. If not, it will complain about an unknown "flavor" of Electron.

**Comment by erickzhao** [comment]
Duplicate of https://github.com/electron/fiddle/issues/1241

Should be in the next release but we've been blocked by CI pipeline issues :'(
